crates/uplc/tests/conformance/builtin/semantics/consByteString/consByteString-01/consByteString-01.uplc

```diff
@@ -1,2 +1,2 @@
 -- the arg overflow'ed over the maxBound :: Word8
-(program 1.0.0 [(builtin consByteString) (con integer 256) (con bytestring #)])
```

Comment:
hmm; is changing the language version here sound / needed? different language versions will have different execution semantics, and we'll need to be able to run both in Amaru, so we should have tests for all versions.